cmd/kubelet/app/server.go

```diff
@@ -418,8 +418,15 @@ func checkPermissions() error {
 	if uid := os.Getuid(); uid != 0 {
 		return fmt.Errorf("Kubelet needs to run as uid `0`. It is being run as %d", uid)
 	}
-	// TODO: Check if kubelet is running in the `initial` user namespace.
+	// Check if kubelet is running in the `initial` user namespace.
 	// http://man7.org/linux/man-pages/man7/user_namespaces.7.html
+	namespace, _ := os.Readlink("/proc/" + strconv.Itoa(os.Getpid()) + "/ns/user")
```

Comment:
This will break the windows usecase ..

Reply:
you‘re right，i’ve add ostype check，thanks！